File: 알고리즘스터디/jisu/programmers/Lv.1/없는 숫자 더하기.js
const solution = (numbers) => {
  let numArr = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0];

  numbers.map((num) => (numArr[num] = 1));

  let result = 0;
  for (let i = 0; i <= 9; i++) {
    if (numArr[i] === 0) result += i;
  }

  return result;
};

File: 알고리즘스터디/jsm/없는숫자더하기.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        boolean[] check = new boolean[10];
        for(int n : numbers) {
            check[n] = true;
        }

        for(int i = 0; i < 10; i++) {
            if(!check[i]) {
                answer += i;
            }
        }
        return answer;
    }
}
